charts/komodor-agent/templates/metrics/configmap-daemon.yaml

```diff
@@ -33,5 +33,20 @@ data:
       namedrop = ["kubernetes_system_container"]
       [inputs.kubernetes.tags]
         measure_type = "usage"
+        
+    [[inputs.kube_inventory]]
+      ## URL for the Kubernetes API
+      url = {{ .Values.communications.apiServerUrl | quote }}
+      url_kubelet = "https://$NODE_IP:10250"
```

Comment:
Is this port always 10250? Don't we need it to be configurable?

Reply:
it's the kubelet port, I guess it won't be changed soon